wandb/sdk/wandb_run.py

```diff
@@ -1467,47 +1467,69 @@ def _datatypes_callback(self, fname: str) -> None:
         files: FilesDict = dict(files=[(GlobStr(glob.escape(fname)), "now")])
         self._backend.interface.publish_files(files)
 
-    def _visualization_hack(self, row: dict[str, Any]) -> dict[str, Any]:
+    def _process_chart(self, chart: Visualize | CustomChart, key: str) -> None:
+        """Update the run config with the chart's config value and key."""
+        chart.set_key(key)
+        self._config_callback(
+            val=chart.spec.config_value,
+            key=chart.spec.config_key,
+        )
```

Comment:
i don't think we need this function for 2 lines, just repeat it, it is more readable anyways

Reply:
why did you resolve this? i really don't think the `Run` object needs another function for this

Reply:
Oops you are right, I moved the logic back in the original function but forgot to delete this 🤦‍♂️